DEPS

```diff
@@ -1,7 +1,7 @@
 use_relative_paths = True
 
 deps = {
-  "vendor/adblock_rust_ffi": "https://github.com/brave/adblock-rust-ffi.git@89127a30655eaf54cf73794309846084ea8b91b9",
+  "vendor/adblock_rust_ffi": "https://github.com/brave/adblock-rust-ffi.git@6dad8c1b47c1b866fa1b1c19e36da6eb5797a524",
```

Comment:
Before this gets merged please merge this: 
https://github.com/brave/adblock-rust-ffi/pull/7

And then update this changeset ref. As I only want to commit the ref here for things in master on the ffi repo.